modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -70,13 +70,13 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-. Add `* file:` to `default-src` in the `Content-Security-Policy` meta tag inside the `index.html` file:
+. Add the following code to `index.html` located in `src/` :
 +
 [source,html]
 ----
 <meta http-equiv="Content-Security-Policy" content="default-src * file: 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *; img-src 'self' data: content:;">
 ----
-. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^]:
+. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^] in `src/`:
```

Comment:
Running this in 'src' resulted in this message:

 - " modified:   ../package.json"

Reply:
that's normal for npm